src/JuliusSweetland.OptiKey.Core/UI/ViewModels/MainViewModel.ServiceEventHandlers.cs

```diff
@@ -2279,15 +2280,66 @@ private void HandleFunctionKeySelectionResult(KeyValue singleKeyValue)
                     break;
 
                 case FunctionKeys.Speak:
+                    string textToSpeak = "";
+                    if (!string.IsNullOrEmpty(keyboardOutputService.Text))
+                    {
+                        if (keyboardOutputService.Text.Contains(TranslationService.LICENSE_TEXT))

```

Comment:
Could you please remove the license text concept entirely? I know there's a requirement (https://tech.yandex.com/translate/doc/dg/concepts/design-requirements-docpage/), but I think the UX is wrong. I will contact Yandex before releasing the code to discuss with them. It may be sufficient to give them a shout somewhere else.